src/sidebar/services/test/frame-sync-test.js

```diff
@@ -191,16 +188,18 @@ describe('FrameSyncService', () => {
   /**
    * Simulate a new guest frame connecting to the sidebar.
    *
+   * @param {string} [frameId] - Guest frame ID, or `undefined` for main frame
```

Comment:
Is there a meaningful difference between "frameId" and "sourceId"?

Reply:
They are used interchangeably at present. In principle a frame that has multiple roles (eg. a frame that is both a guest and a host) could use a different ID for each role. That doesn't occur at present.